requirements_linter.txt

```diff
@@ -14,3 +14,4 @@ xmltodict >= 0.2
 pyyaml >= 5.1
 h5py >= 3.2.1
 pillow >= 2.0.0
+jinja2 >= 3.0.1
```

Comment:
why we need `jinja2`？can we just use f-string as template?

Reply:
We need to use Jinja2 to pass parameters and make some judgments.

Reply:
is that possible to move the judgment part to python?